content/post/agile-analytics-p3.md

```diff
@@ -0,0 +1,91 @@
+---
+author: Michael Kaminsky
+title: "Agile Analytics, Part 3: The Adjustments"
+publishDate: "2018-07-01"
+draft: true
+tags: 
+ - agile
+ - scrum
+ - process
+---
+
+
+Agile software engineering practices have become the standard work management tool for modern software development teams. Are these techniques applicable to analytics, or is the nature of research prohibitively distinct from the nature of engineering?
+
+In this post I discuss some adjustments to the scrum methodology to make the process work better for analytics and data science teams.
```

Comment:
In some posts, including your first post in this series, references to teams are capitalized, e.g., `Analytics team`, whereas the term `analytics` by itself isn't. If that's how we want to do it, then I think this should read `Analytics and Data Science teams.`